README.md

```diff
@@ -60,177 +26,103 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
-
-> Describe your project in 1 or 2 sentences.
+# 📖 Capstone-project <a name="about-project"></a>
 
-**[your_project__name]** is a...
+Its a capstone project and this website is about makeup that's all.
 
-## 🛠 Built With <a name="built-with"></a>
+## 🛠 Built With <a name="visual studio">HTML5</a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
-
-<details>
-  <summary>Client</summary>
-  <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
-  </ul>
-</details>
-
-<details>
-  <summary>Server</summary>
-  <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
-  </ul>
-</details>
-
-<details>
-<summary>Database</summary>
-  <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
-  </ul>
-</details>
+- **html**
+- **Css**
+- **Javascript**
 
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
 > Describe between 1-3 key features of the application.
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **It has linters**
+- **It has MIT license**
+- **It has html**
+- **It has CSS**
+- **It has javascript**
+- **Responsive design**
+- **Feature speakers part is fully made dynamically by JavaScript**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
-
-## 🚀 Live Demo <a name="live-demo"></a>
-
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://google.com)
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] You did well deploying the project using Netlify, but based on the project requirement, you should use GitHub pages to deploy the project. In Microverse and in real-world work, developers are expected to follow the client's requirements before adding their creativity to the project. 

Kindly fix that.